requirements.in

```diff
@@ -7,7 +7,7 @@ flatpak-module-tools >= 0.11;python_version>="3.9"
 jsonschema
 PyYAML
 ruamel.yaml
-git+https://github.com/containerbuildsystem/osbs-client@be934592d8c2a90c186bdab619a1dffb71cf62c1#egg=osbs-client
+git+https://github.com/rcerven/osbs-client@f458cb5059dc6ab0e53189217ed1cbd94066b245#egg=osbs-client
```

Comment:
Even for testing, you will need to update this in requirements.txt as well (ideally by running `make pip-compile`)

Reply:
Cachito requires the exact commit, and I think `pip-compile` doesn't support resolving branches to commits so it already has to be a commit in requirements.in